tempodb/tempodb.go

```diff
@@ -633,7 +633,7 @@ func createLegacyCache(cfg *Config, logger gkLog.Logger) (cache.Cache, []cache.R
 			(cfg.Search.CacheControl.ColumnIndex ||
 				cfg.Search.CacheControl.Footer ||
 				cfg.Search.CacheControl.OffsetIndex) {
-			return nil, nil, errors.New("no legacy cache configured, but cache_control is enabled. Please use the new top level cache configuration.")
+			return nil, nil, errors.New("no legacy cache configured, but cache_control is enabled. Please use the new top level cache configuration")
```

Comment:
It looks like CI is failing because we test for this string.  Since we have two sentences here, the period at the end make some sense to me.  What do you think?

Reply:
ok